packages/contact-manager-rest-api/__tests__/database/databaseService.ts

```diff
@@ -0,0 +1,36 @@
+import Debug from 'debug'
```

Comment:
ssi-agent has a class DataSources that handles all of this these days.

Reply:
changed debug to console logs

Reply:
changed this to use DataSources from `@sphereon/ssi-sdk.agent-config` instead and therefore removed the extra class `databaseService`